README.md

```diff
@@ -19,7 +19,7 @@ First add the dependency of the cloudstater starter to your project. Here an Mav
 <dependencies>
     <dependency>
         <groupId>io.cloudstate</groupId>
-        <artifactId>springboot-support</artifactId>
+        <artifactId>cloudstate-springboot-support</artifactId>
```

Comment:
was this necessary to standardize the name of the artifact similar to that of cloudstate-java-support?

Reply:
Yes, just wanted to keep the same naming convention. I'm not sure whether we need `-support` on these, but that's the pattern currently.